src/components/mobile-nav-item/types.ts

```diff
@@ -0,0 +1,21 @@
+import { ReactNode } from 'react'
+
+type CommonMobileNavItemProps = {
+  label: string
+  isActive?: boolean
+  hasBadge?: boolean
+}
+
+export interface MobileNavItemSimpleProps extends CommonMobileNavItemProps {
+  children?: never
+  isDefaultExpanded?: never
+  href: string
```

Comment:
**suggestion:** I think we need to support `onClick` too?

Reply:
>  I think we need to support onClick too?

good catch on this